lti_consumer/utils.py

```diff
@@ -16,21 +16,21 @@ def lti_1p3_enabled():
     """
     Returns `true` if LTI 1.3 integration is enabled for instance.
     """
-    return settings.FEATURES.get('LTI_1P3_ENABLED', False) is True  # pragma: no cover
+    return settings.FEATURES.get('LTI_1P3_ENABLED', True) is True  # pragma: no cover
 
 
 def lti_deeplinking_enabled():
     """
     Returns `true` if LTI Advantage deep linking is enabled for instance.
     """
-    return settings.FEATURES.get('LTI_DEEP_LINKING_ENABLED', False) is True  # pragma: no cover
+    return settings.FEATURES.get('LTI_DEEP_LINKING_ENABLED', True) is True  # pragma: no cover
 
 
 def lti_nrps_enabled():
     """
     Returns `true` if LTI NRPS is enabled for instance.
     """
-    return settings.FEATURES.get('LTI_NRPS_ENABLED', False)  # pragma: no cover
+    return settings.FEATURES.get('LTI_NRPS_ENABLED', True)  # pragma: no cover
```

Comment:
We need to coordinate this with a change to edx.org settings to keep this disabled.

Reply:
Makes sense (I can't review edX-internal PR's though - so I won't be able to help with this).

Reply:
Right, and it doesn't have to happen before merging this, just before tagging to make a new release on PyPI.

Reply:
@nedbat Is there need to gate this feature? The default configuration doesn't transmit any PII.